ci/build_test_OnCommit.groovy

```diff
@@ -0,0 +1,89 @@
+def image_build_needed = "false"
+def client_test_needed = "false"
+def shortCommit = ""
+
+pipeline {
+    agent {
+      label 'ovmsbuilder'
+    }
+    stages {
+        stage('Configure') {
+          steps {
+            script {
+              shortCommit = sh(returnStdout: true, script: "git log -n 1 --pretty=format:'%h'").trim()
+              echo shortCommit
+              echo sh(script: 'env|sort', returnStdout: true)
+              def git_diff = ""
+              if (env.CHANGE_ID){ // PR - check changes between target branch
+                sh 'git fetch origin ${CHANGE_TARGET}'
+                git_diff = sh (script: "git diff --name-only \$(git merge-base FETCH_HEAD HEAD)", returnStdout: true).trim()
+                println("git diff:\n${git_diff}")
+              } else {  // branches without PR - check changes in last commit
+                git_diff = sh (script: "git diff --name-only HEAD^..HEAD", returnStdout: true).trim()
+              }
+              def matched = (git_diff =~ /src|third_party|(\n|^)Dockerfile|(\n|^)Makefile|\.c|\.h|BUILD|WORKSPACE|(\n|^)rununittest\.sh/)
```

Comment:
This does not include all needed matches (bazel.rc, .bzl) files. Why we don't use blacklist instead?

Reply:
Also rebuild will not trigger if you will add new licesne to `release_files`.

Reply:
@atobiszei  indeed but the license file has no impact on the image behavior and oncommit testing results. In my opinion there is not point in building the image and functional tests only if the license changes